bridge_adaptivity/module/templates/module/collection_detail.html

```diff
@@ -57,14 +57,20 @@
                     {% bootstrap_button "" size='sm' icon='trash' %}
                   </a>
                   {% ifnotequal activity.atype 'G' %}
+                    {% ifnotequal activity.order 0 %}
                     <a class="move-up pull-right"
                        href="{% url 'module:activity-move' pk=activity.pk direction='up' %}">
                       {% bootstrap_button "" size='sm' icon='chevron-up' %}
                     </a>
-                    <a class="move-down pull-right"
-                       href="{% url 'module:activity-move' pk=activity.pk direction='down' %}">
-                      {% bootstrap_button "" size='sm' icon='chevron-down' %}
-                    </a>
+                    {% endifnotequal %}
+                    {% if not forloop.last %}
```

Comment:
Why do we check last in forloop and lastpre in different condition statements you can use one with the OR operator.

Reply:
actually I tried... It didn't work...